splice_machine/src/main/java/com/splicemachine/derby/utils/SpliceAdmin.java

```diff
@@ -1280,24 +1280,70 @@ public static void SYSCS_GET_GLOBAL_DATABASE_PROPERTY(final String key,final Res
         resultSet[0] = new EmbedResultSet40(conn, resultsToWrap, false, null, true);
     }
 
-    public static void SYSCS_SET_GLOBAL_DATABASE_PROPERTY(final String key,final String value) throws SQLException{
+    static boolean databasePropertyExists(final String key) {
+        SConfiguration config=EngineDriver.driver().getConfiguration();
+
+        // get configurations from SConfigurationImpl and Hadoop ConfigurationSource
+        Map<String,Object> configRootMap = config.getConfigMap();
+        if(configRootMap != null && configRootMap.getOrDefault(key, null) != null)
+            return true;
+
+        // get configuration fields from class com.splicemachine.db.iapi.reference.Property
+        for (Field field : com.splicemachine.db.iapi.reference.Property.class.getFields()) {
+            try {
+                if(field.getType().equals(String.class)
+                        && ((String)field.get(null)).compareTo(key) == 0 ) {
+                    return true;
+                }
+            } catch (IllegalAccessException e) {
+                // continue
+            }
+        }
+        return false;
+    }
+
+    public static void SYSCS_SET_GLOBAL_DATABASE_PROPERTY(final String key, final String value,
+                                                          final ResultSet[] resultSet) throws SQLException{
         try {
             LanguageConnectionContext lcc = ConnectionUtil.getCurrentLCC();
             SpliceTransactionManager tc = (SpliceTransactionManager)lcc.getTransactionExecute();
             DataDictionary dd = lcc.getDataDictionary();
+            lcc.getLastActivation().addWarning(new SQLWarning("WARNING!!!\n"));
```

Comment:
Why always add a warning?

Reply:
ah sorry i forgot this one. i tried to print warnings, but this doesn't work currently when doing CALLs for some reason